common/autoware_lanelet2_utils/test/conversion.cpp

```diff
@@ -214,8 +217,8 @@ TEST(ArtificialLaneletObjectConstruction, ConstLaneletConstruct)
   {
     const auto opt =
       autoware::experimental::lanelet2_utils::create_safe_lanelet(left_points, right_points);
-    EXPECT_TRUE(opt.has_value()) << "BasicPoint3d can construct normally.";
-    const auto ll = *opt;
+    ASSERT_TRUE(opt.has_value()) << "BasicPoint3d can construct normally.";
+    const auto ll = opt.has_value() ? *opt : lanelet::ConstLanelet{};
```

Comment:
I guess it is not recommended to initialize with empty lanelet (`lanelet::ConstLanelet{}`) because it is assumed to access inside of for-loop. Also as it is guaranteed to have value thanks to `ASSERT_TRUE(opt.has_value())`, my idea is:
```suggestion
    const auto ll = *opt;
```

Reply:
Thank you for your comment! It was `ll = *opt` before, but I changed it temporary to pass the clang-tidy, I will change it back!

Reply:
Sorry, it looks `ASSERT_TRUE(opt.has_value())` check not working, ... .